developer-guide.md

```diff
@@ -70,4 +70,4 @@ The [experimental feature tracker](available-trials.md) lists all of the current
     - example.com
   - To ensure that an experimental feature is not enabled too broadly, there are some additional checks on requests for subdomain tokens. Specifically, tokens will not be issued for origins found in the [Public Suffix List](https://publicsuffix.org/).
   - Subdomains do not apply to IP addresses. Tokens issued for IP addresses will only allow exact matching on origin, as before.
-  - You can request a subdomain token by selecting the appropriate checkbox on the [trial signup form](http://bit.ly/OriginTrialSignup).
+  - You can request a subdomain token by selecting the appropriate checkbox on the trial signup form.
```

Comment:
I think the main point is to let users know that the signup form is where to ask for a subdomain token. I think instructions can be less specific about the UI, in case we change the form fields (e.g. use a toggle instead of a checkbox). 

```suggestion
  - You can request a subdomain token by filling out the appropriate field on the trial signup form.
```